charts/lakefs/templates/_env.tpl

```diff
@@ -25,6 +25,10 @@ env:
   - name: LAKEFS_GATEWAYS_S3_FALLBACK_URL
     value: http://localhost:7001
   {{- end }}
+  {{- if .Values.local_cache_volume }}
+    name: LAKEFS_COMMITTED_LOCAL_CACHE_DIR
+    value: /lakefs/cache
```

Comment:
so what happens if the user doesn't specify a volume? this path will not exist and lakeFS initialization will fail?

Reply:
No, in this case this `if` will return false - it will not fail.
(However, I have a typo in the parameter name, fixing)